tiny_tds.gemspec

```diff
@@ -24,8 +24,8 @@ Gem::Specification.new do |s|
   s.add_development_dependency 'mini_portile2', '~> 2.5.0'
   s.add_development_dependency 'rake', '~> 13.0.0'
   s.add_development_dependency 'rake-compiler', '~> 1.2'
-  s.add_development_dependency 'rake-compiler-dock', '~> 1.4.0'
```

Comment:
What changed here?

Reply:
* v1.5: Support for compiling gems against musl (Alpine's "glibc" library)
* v1.6: Base images are now based on Ubuntu, therefore some libraries were upgraded.
* v1.7: Support for compiling gems against Ruby 3.4

Reply:
The release notes [here](https://github.com/rake-compiler/rake-compiler-dock/releases/tag/v1.6.0) say the minimum glibc version was incremented; does that mean _our_ minimum version of linux/glibc is also bumped for our linux releases? or is that just for the build system for windows?

Reply:
that is just for the build system on Windows.